app/src/main/java/app/revanced/integrations/patches/components/AdsFilter.java

```diff
@@ -76,7 +76,8 @@ public AdsFilter() {
         final var infoPanel = new StringFilterGroup(
                 SettingsEnum.HIDE_HIDE_INFO_PANELS,
                 "publisher_transparency_panel",
-                "single_item_information_panel"
+                "single_item_information_panel",
+                "shorts_info_panel_overview"
```

Comment:
How about adding this here, as well as in `ShortsComponetsFilter`? The reason for this is that this component is related to shorts. The question would be, if the user disables the switch to hide the component in the `shorts-componets` patch, it will not make any change unless they change it in the `hide-ads` patch toggle, so the toggles have to be able to turn each other off. Maybe a separate toggle can be added and used with the same name in both patches so that toggling one toggle toggles the other too. Before implementing, please let me know what you think about this, so that we can discuss how we want to implement it.

Reply:
> but would be a bunch of work now

It is as simple as adding a separate preference screen with two preferences. One that hides the info panel, another that hides the shorts info panel. The same preference key for the shorts info panel switch can then be used for a switch in the shorts components preference screen; toggling either preference will turn both toggles on or off and successfully satisfy the problem semantically. 

On an unrelated note, currently, litho patches are hard to separate. If you think about it, the info panel is not part of hiding ads, yet it is added through that patch. I have not come up with a proper solution for this yet.